grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceLogNameTest.java

```diff
@@ -103,9 +103,11 @@ void logName() {
         client.emptyCall(Empty.newBuilder().build());
 
         final RequestLog log = capturedCtx.log().partial();
-        assertThat(log.serviceName()).isEqualTo(TestServiceGrpc.SERVICE_NAME);
-        assertThat(log.name()).isEqualTo("EmptyCall");
-        assertThat(log.fullName()).isEqualTo(TestServiceGrpc.getEmptyCallMethod().getFullMethodName());
+        final String framedGrpcServiceName = FramedGrpcService.class.getName();
+        assertThat(log.serviceName()).isEqualTo(framedGrpcServiceName);
+        final String emptyFullMethodName = TestServiceGrpc.getEmptyCallMethod().getFullMethodName();
+        assertThat(log.name()).isEqualTo(emptyFullMethodName);
+        assertThat(log.fullName()).isEqualTo(framedGrpcServiceName + "/" + emptyFullMethodName);
```

Comment:
I don't think we can change this test case.
The default service name of a gRPC service should be gotten from  `StubName` + `Grpc.SERVICE_NAME`

Reply:
Agreed. Could we update all RPC services (gRPC and Thrift) so they set the correct gRPC/Thrift service names via `RequestLogBuilder.name(String, String)`?